e2e/tests/integrations/create-integration-as-integrator.test.ts

```diff
@@ -0,0 +1,28 @@
+import { expect, test } from "@playwright/test";
+
+test("As an integrator I can create a new integration", async ({ page }) => {
+  await page.goto("/nl/integraties");
+  await expect(page.locator("h2").getByText("Mijn integraties")).toBeVisible();
+
+  await page.getByRole('main').getByRole('link', { name: 'Integratie toevoegen' }).click();
+  await page.getByLabel('Naam integratie').click();
```

Comment:
Is using `locator` more robust?

Reply:
Here are the best practices of playwright: 
https://playwright.dev/docs/best-practices#best-practices

I think this is an ok selector.
I think we should avoid class based selectors:

> // 👎
page.locator('button.buttonIcon.episode-actions-later');